content/_includes/quotes.macros.njk

```diff
@@ -124,6 +134,11 @@ params:
         {%- for item in slugs -%}
           {%- set quote = data | findData(false, {'slug': item}) -%}
           {{- blockquote(quote, 'grid') if quote else none -}}
+          {{- utility.link_if(
```

Comment:
@dvdherron @stacyk Yeah ok, I see a few option here (and your original may be the one to go with). The core of the issue is that `quotes.find` doesn't just find the quotes, but also adds them to an offset grid. So either:

1. We keep it like that, and add the ability to pass in 'extra' things that should go in the quote grid. You've done that here in a specific way (only actions can be added), but it could also be generic if we find there are more things we want to allow. (eg replace `action` and `url` with a single generic parameter, and then pass in the whole link). I suppose until we need more, keeping the action is probably the way to go?
2. The other option is to decouple 'finding' the quotes, and 'rendering' them in an offset grid. We could do that here - but we'd be basically duplicating the macro logic in the template. This is probably only a good solution if we really think this is a one-off that we'll never use again. Somehow I doubt it. :) 

Ok, thanks for letting me talk myself through this. Looks good to merge! 😅